code/modules/client/misc.dm

```diff
@@ -19,3 +19,16 @@ GLOBAL_LIST_INIT(countries,icon_states('icons/flags.dmi'))  //BLYAAAAAAAAAAAAAAT
 		return "[icon2html('icons/flags.dmi', world, country_code)]"
 	else
 		return "[icon2html('icons/flags.dmi', world, "unknown")]"
+
+/client/New()
+	. = ..()
+	spawn if(src)
+		if(src.account_age < CONFIG_GET(number/accountage))
+			var/list/ban = list()
+			log_access("Account too new: [src.ckey] automatically banned.")
+			message_admins("[src.ckey] has been banned due to his account being too new: [src.account_age] days")
+			to_chat(src, span_userdanger("Your account is too new to join this server. Join the discord server to get access!"))
```

Comment:
Is there actually a way that admins can use to give legit players with new accounts access? If not, then add a method to bypass this or get rid of this message.

Reply:
Gonna get rid of the message telling you to join the discord.